backend/bitboxbase/handlers/handlers.go

```diff
@@ -462,3 +464,16 @@ func (handlers *Handlers) getServiceInfoHandler(_ *http.Request) (interface{}, e
 		"serviceInfo": baseInfo,
 	}, nil
 }
+
+func (handlers *Handlers) getUpdateInfoHanlder(_ *http.Request) (interface{}, error) {
```

Comment:
Hanlder -> Handler

Reply:
Good catch, darn typos. Sometimes better to copy-paste :D